spec/openapi.yaml

```diff
@@ -7090,6 +7090,77 @@ paths:
 #
 # Tags API
 #
+  /tags-api/v2/accounts/{accountUid}/tags:
+    post:
+      summary: List tags within a account
+      description: List / search tags within a account.
+      tags:
+        - Tags
+      operationId: getTagsListByAccount
+
+      parameters:
+        - description: A unique identifier for an account for search.
+          in: path
+          name: accountUid
+          required: true
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/json:
+            schema:
+              properties:
+                projectIds:
+                  items:
+                    description: String hashcode
```

Comment:
Update description

Reply:
Done

Reply:
for accountUID please change description from “A unique identifier for an account for search.” to  “The account’s unique identifier. This can be found in the Smartling Dashboard under Account Settings > API.”
projectID array change description to:
Optional array of unique project identifiers. This can be found in the Smartling Dashboard under Account Settings > API.

For tagMask description: “Search filter.  Will return tags with partial match and is case insensitive.”

Reply:
Done